charts/manifests/Chart.yaml

```diff
@@ -22,4 +22,4 @@ annotations:
   artifacthub.io/prerelease: "true"
   artifacthub.io/license: Apache-2.0
   artifacthub.io/changes: |
-    - "[Fixed]: Pod missing name in metadata"
+    - add newlines to prevent parse errors
```

Comment:
For the next Contribution: We tag each change with the corresponding change category. Template was newly added to https://github.com/bedag/helm-charts/blob/master/CONTRIBUTING.md#changelog. So would be:

```
- "[Fixed]: add newlines to prevent parse errors"
```